react-app/src/components/PortfolioScreen/PortfolioScreenAPI.ts

```diff
@@ -114,144 +112,3 @@ export const usePortfolioQuery = (): {
     fetch,
   };
 };
-
-export const useStakesQuery = (): {
-  requestState: RequestState<Stake[]>;
-  fetch: (address?: string) => Promise<void>;
-  order: ColumnOrder;
-  setOrder: (order: ColumnOrder) => void;
-} => {
-  const [requestState, setRequestState] =
-    useState<RequestState<Stake[]>>(RequestStateInitial);
-
-  const wallet = useWallet();
-  const stakingAPI = useStakingAPI();
-  const distribution = useDistributionAPI();
-  const { query } = useQueryClient();
-
-  const [order, setOrder] = useState({
-    id: "name",
-    direction: "asc" as ColumnOrder["direction"],
-  });
-
-  const isValidAddress = useCallback(
-    async (address: string) => {
-      try {
-        const account = await query.auth.account(address);
-        return account != null;
-      } catch {
-        return false;
-      }
-    },
-    [query]
-  );
-
-  const fetchStakes = useCallback(
-    async (address: string) => {
```

Comment:
why did we add this in the previous commit and then remove in this commit? can we fix up for better readability